include/tvm/tir/usmp/utils.h

```diff
@@ -163,7 +162,7 @@ class BufferInfoAnalysis : public ObjectRef {
  * \brief The pool allocation produced after the USMP algorithm
  */
 struct PoolAllocationNode : public Object {
-  /*! \brief The assigned PoolInfo object */
+  /*! \brief The assigned WorkspacePoolInfo object */
```

Comment:
comment doesn't match the type of the attr

Reply:
amended